sources/networkmanager/site.go

```diff
@@ -0,0 +1,139 @@
+package networkmanager
+
+import (
+	"context"
+	"fmt"
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/networkmanager"
+	"github.com/aws/aws-sdk-go-v2/service/networkmanager/types"
+	"github.com/overmindtech/aws-source/sources"
+	"github.com/overmindtech/sdp-go"
+	"strings"
+)
+
+func sitesGetFunc(ctx context.Context, client NetworkmanagerClient, scope string, input *networkmanager.GetSitesInput) (*sdp.Item, error) {
+	if input == nil || input.GlobalNetworkId == nil {
+		return nil, &sdp.QueryError{
+			ErrorType:   sdp.QueryError_NOTFOUND,
+			ErrorString: "GetSitesInput.GlobalNetworkId param is mandatory",
+		}
+	}
+	out, err := client.GetSites(ctx, input)
+	if err != nil {
+		return nil, err
+	}
+	if len(out.Sites) == 0 {
+		return nil, &sdp.QueryError{
+			ErrorType:   sdp.QueryError_NOTFOUND,
+			ErrorString: fmt.Sprintf("site with ID %s not found", input.SiteIds[0]),
+		}
+	}
+	if len(out.Sites) != 1 {
+		return nil, fmt.Errorf("got %d sites, expected 1", len(out.Sites))
+	}
+
+	site := out.Sites[0]
+
+	attributes, err := sources.ToAttributesCase(site, "tags")
```

Comment:
Below we are creating a custom Id so that a sit can be fetched using `{globalNetworkId}/{siteId}` which is a good idea if the side doesn't have a unique ID it's own. But if we are doing that we need to make sure that we are also creating that attribute and adding it here since it won't be in the response

Reply:
I got the idea, but can't find a good example for it. Could you find a good example for this case please?